src/components/user/SignUpForm/styles.js

```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+  signUp: {
+    width: 230
```

Comment:
Mmm why do you need this fixed width? A horizontal padding would be better

Reply:
I just copied the style from the login form. Should I change both?

Reply:
Yeah I think. We shouldn't use fixed width unless necessary, you can achieve the same and better (responsive) results with an horizontal padding here

Reply:
Ok, I will, but in another PR fixing both styles. Sounds ok?

Reply:
Sounds good!

Reply:
For me, it's okay if you do it on this PR.